openhands/runtime/impl/remote/remote_runtime.py

```diff
@@ -90,9 +90,8 @@ def __init__(
         self.runtime_url: str | None = None
 
     async def connect(self):
-        await call_sync_from_async(self._start_or_attach_to_runtime)
         try:
-            await call_sync_from_async(self._wait_until_alive)
+            await call_sync_from_async(self._start_or_attach_to_runtime)
```

Comment:
We are already calling `self._wait_until_alive` inside `self._start_or_attach_to_runtime` - so probably not useful to call it again here